src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_has_expired.cs

```diff
@@ -39,10 +39,30 @@ public Task Handle(MyMessage message, IMessageHandlerContext context)
                     return Task.FromResult(0);
                 }
             }
+
+            class DelayReceiverFromStarting: IWantToRunWhenBusStartsAndStops
+            {
+                /// <summary>
+                /// Method called at startup.
+                /// </summary>
+                public async Task Start(IBusSession session)
+                {
+                    await session.SendLocal(new MyMessage());
+                    await Task.Delay(TimeSpan.FromSeconds(5));
```

Comment:
the reason i ask is because i recently shaved off 30% of the acceptance tests total execution time by just finding overly generous wait times for things to happen.